docs/moderator-handbook.md

```diff
@@ -187,11 +193,15 @@ In exceptional circumstances, it may be appropriate for a moderator to fork a di
 
 Again, if you have any problems or questions, make a post with your actions in the `"Staff"` category, and tag another moderator if you want them to review your moderating actions.
 
+### Dealing with Posted Solutions
+
+When a user posts a solution to any of the questions, remove it and either, kindly let them know that solutions are not permitted on the forum in your own words or, use the **Solution Instead of Help** "Canned reply".
+
 ### Underage Users
 
 Our [Terms of Service](https://freecodecamp.org/terms) require that freeCodeCamp users be at least 13 years of age. If a user reveals that they are under the age of 13, send them the below message and delete their forum account (if deletion is not available, suspending the account is sufficient).
 
-**Email `support[at]freecodecamp.org` to delete the user's freeCodeCamp account as well.**
+**Email `support[at]freecodecamp.org` to delete the user's freeCodeCamp account as well (providing the email address and user id to support).**
```

Comment:
The user id is generated by Discourse, and is in no way connected to the `/learn` account. So, it is often more useful to provide the username which might match the `/learn` username.

```suggestion
**Email `support[at]freecodecamp.org` to delete the user's freeCodeCamp account as well (providing the email address and username of the offending forum account).**
```